src/plugins/timeline/TimelineViewLayout.vue

```diff
@@ -92,6 +94,10 @@ export default {
       this.composition.on('reorder', this.reorder);
       this.composition.load();
     }
+
+    this.handleContentResize = _.debounce(this.handleContentResize, 500);
```

Comment:
We can use this guy now instead of importing lodash: https://github.com/nasa/openmct/blob/master/src/utils/debounce.js